### DISCUSSION TITLE
name of robots for the finals

### DISCUSSION THREAD

**Discussion by ijnek** [cyberbotics/wrestling#111] [Q&A] [answered]
@omichel
My code was working when I was the participant, but failing when I was the oppoonent. Investigating this, it turned out that using the ROS 2 webots controller, the topics get namespaced by the robot name (as defined in wrestling.wbt), and I was only listening to ROS 2 topics from the participant.

I can get around this, by remapping both /opponent/... and /participant/... topics onto a topic with the same name, but coulld you please clarify that in the finals, the name of the wrestlers willl **only be "opponent" or "participant"**?

**Comment by omichel** [comment, marked as answer]
Hi,

Sorry for this trouble. However, I can confirm that during the finals the names of the wrestlers will be only either **"opponent"** or **"participant"**.
